features/supplier/view_and_edit_g_cloud_services.feature

```diff
@@ -9,13 +9,7 @@ Background:
   And we accept that suppliers application to the framework
   And that supplier returns a signed framework agreement for the framework
   And that supplier has a service on the cloud-support lot
-  When I visit the /suppliers page
-  # The following step only works by virtue of there only being a single service for this supplier - multiple services on
-  # multiple frameworks will cause multiple "View services" links to be present
-  And I click 'View services'
-  Then I am on the 'G-Cloud 10 services' page
-  When I click 'Test cloud support service'
-  Then I am on the 'Test cloud support service' page
+  Then that supplier visits their page for that service
```

Comment:
I'm a bit reluctant to remove the `Then I am on the 'G-Cloud 10 services' page` step entirely - I think this is the only scenario where we visit the page with the supplier's list of services. We should follow the journey that the user would take (I know this isn't consistently done in other scenarios!) wherever possible.

What if we changed the step to be `Then I am on the 'Your G-Cloud' page` combined with `And I see the page's h1 ends in 'services'`?